.github/workflows/integration-test.yml

```diff
@@ -13,50 +13,57 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # TODO: ['14.x', '16.x']
-        node-version: [14.x]
+        node-version: [18.x]
     steps:
     - name: Checkout dapp
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     # Select a branch on agoric-sdk to test against by adding text to the body of the
     # pull request. For example: #agoric-sdk-branch: zoe-release-0.7.0
-    # The default is 'master'
+    # The default is 'NONE'
     - name: Get the appropriate agoric-sdk branch
       id: get-branch
-      uses: actions/github-script@0.9.0
+      uses: actions/github-script@v6
       with:
         result-encoding: string
         script: |
           const { body = '' } = context.payload.pull_request || {};
           const regex = /.*\#agoric-sdk-branch:\s+(\S+)/;
           const match = regex.exec(body);
-          const agoricSdkBranch = match && match[1] || 'master';
+          const agoricSdkBranch = match && match[1] || 'NONE';
           console.log(agoricSdkBranch);
           return agoricSdkBranch;
 
     - name: Checkout agoric-sdk
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+      if: ${{ steps.get-branch.outputs.result != 'NONE' }}
       with:
         repository: Agoric/agoric-sdk
         path: agoric-sdk
         ref: ${{steps.get-branch.outputs.result}}
   
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
 
     - name: Setup and link agoric-sdk packages
+      if: ${{ steps.get-branch.outputs.result != 'NONE' }}
       run: |
         yarn install
         yarn build
         yarn link-cli ~/bin/agoric
         echo "/home/runner/bin" >> $GITHUB_PATH
       working-directory: ./agoric-sdk
 
-    - name: agoric install
-      run: agoric install
+    - name: install packages
+      run: |
+        if ${{ steps.get-branch.outputs.result == 'NONE' }}; then
+          yarn install
+          echo "AGORIC_CMD=[\"$PWD/node_modules/.bin/agoric\"]" >> $GITHUB_ENV
```

Comment:
Is there a way we can avoid this workaround? What is it needed for anyway?

Reply:
The agoric-cli/tools integration test needs to know how to find the CLI, and it defaults to using the PATH.  I didn't want to munge with the `$GITHUB_PATH` file, so I used the same `$AGORIC_CMD` environment variable that the `agoric-sdk/scripts/registry.sh` uses.

Reply:
Oh I didn't see this used anywhere, so I thought it was a magic env